src/main/java/com/eternalcode/parcellockers/user/User.java

```diff
@@ -0,0 +1,37 @@
+package com.eternalcode.parcellockers.user;
+
+import org.bukkit.entity.Player;
+
+import java.util.Set;
+import java.util.UUID;
+
+public class User {
+
+    private final String name;
+    private final Set<UUID> parcels;
+    private final UUID uuid;
+
+    public User(Player player, Set<UUID> parcels) {
+        this.name = player.getName();
+        this.parcels = parcels;
+        this.uuid = player.getUniqueId();
+    }
+
+    protected User(UUID uuid, String name, Set<UUID> parcels) {
```

Comment:
czemu to jest protected?

Reply:
Bo konstruktor w tej formie jest używany tylko wewnątrz package'a `user`